fava/help/extensions.md

```diff
@@ -29,3 +51,15 @@ Called after metadata (`key: value`) has been added to an `entry`.
 ### `after_insert_entry(entry)`
 
 Called after an `entry` has been inserted.
+
+---
+
+### `report_template()`
```

Comment:
Remove, i.e., update with documentation on where the template should be placed.

Reply:
Done, documentation updated at https://github.com/beancount/fava/pull/872/commits/1b282eadaa5778e92d9caef27003b9ed443e0112#diff-a254dd640147f6c3f0aa361a8c736821